packages/big-design/src/components/TableNext/mixins/display/types.ts

```diff
@@ -0,0 +1,16 @@
+import { ThemeInterface } from '@bigcommerce/big-design-theme';
+import { FlattenSimpleInterpolation } from 'styled-components';
+
+import { ResponsiveProp } from '../../../../types';
+
+type TableColumnDisplayProp = ResponsiveProp<'table-cell' | 'none'>;
+
+export type TableColumnDisplayProps = Partial<{
+  display: TableColumnDisplayProp;
+}>;
```

Comment:
I agree, we could use `?` instead of `Partial`. I just gonna change `type` to `interface` since it's causing eslint errors `Use an `interface` instead of a `type`

would be like this:
```
export interface TableColumnDisplayProps {
  display?: TableColumnDisplayProp;
}
```